src/Razor/Microsoft.NET.Sdk.Razor/test/IntegrationTests/ProjectDirectory.cs

```diff
@@ -120,12 +115,13 @@ void CopyDirectory(DirectoryInfo source, DirectoryInfo destination, bool recursi
                 }
             }
 
-            void SetupDirectoryBuildFiles(string solutionRoot, string binariesRoot, string testAppsRoot, string projectDestination)
+            void SetupDirectoryBuildFiles(string repoRoot, string binariesRoot, string testAppsRoot, string projectDestination)
             {
                 var beforeDirectoryPropsContent =
 $@"<Project>
   <PropertyGroup>
-    <SolutionRoot>{solutionRoot}</SolutionRoot>
+    <RepoRoot>{repoRoot}</RepoRoot>
```

Comment:
IMO, these probably shouldn't be needed. We can set these via command line args when invoking msbuild.